cross/psutil-3.2.1/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME = psutil
+PKG_VERS = 3.2.1
```

Comment:
Fyi, we'll probably include psutil in the next version of the Python package(s), which means some shuffling around of cross packages (`cross/psutil` will be latest available, which would be 3.3.0, if other versions are required, those can be added as separate cross packages with version appended)